cev_eris.dme

```diff
@@ -1920,22 +1920,15 @@
 #include "code\modules\random_map\noise\noise.dm"
 #include "code\modules\random_map\noise\ore.dm"
 #include "code\modules\random_map\noise\tundra.dm"
-#include "code\modules\reagents\Chemistry-Colours.dm"
```

Comment:
These changes are bad i think.
Those filenames made those files easy to find with a direct search. 

This change just makes that worse, doesn't really make it more understandable
What's the intention here?

Reply:
That names make it harder to find. You still can find folder with "Chem" and them jump to reqed file with one button "C" for colors, "H" for holdes, "L" for logging and so on.